### ISSUE TITLE
No space is added after REF= 

### ISSUE THREAD

**Issue by Roald87** [Barteling/STweep#16]
When formatting the following code, no space it placed after `REF=`. I expected it to behave the same as with `:=`.

# Example
```
// in
THIS^.var1 REF=var2;

//expected
THIS^.var1 REF= var2;

// current STweep result
THIS^.var1 REF=var2;
```
STweep v2.0.0-beta
[settings.txt](https://github.com/Barteling/STweep/files/5879044/settings.txt)

**Comment by Barteling** [comment]
Can do you a quick check where you set the new "Spaces around assignment operator in invocation call" setting to true?

**Comment by Roald87** [comment]
Do you mean this one? I can't find the option which is called exactly what you said.

![image](https://user-images.githubusercontent.com/20192767/105974921-abdde280-608e-11eb-99cf-5c5a8cba7157.png)

**Comment by Barteling** [comment]
hmm yes that's the one

**Comment by Barteling** [comment]
Actually I cannot reproduce it with STweep-2.0.0-Beta , is there something else going on? is the rest of the file formatted ok? no compiler errors?

**Comment by Roald87** [comment]
Sorry the example is indeed working. I updated the example to a non working one.

**Comment by Barteling** [comment]
Fixed in release 2.0.1.
https://www.stweep.com/Releases/
